src/core/hle/service/fs/fs_user.cpp

```diff
@@ -752,6 +752,42 @@ void FS_USER::ObsoletedDeleteExtSaveData(Kernel::HLERequestContext& ctx) {
               static_cast<u32>(media_type));
 }
 
+void FS_USER::GetSpecialContentIndex(Kernel::HLERequestContext& ctx) {
+    IPC::RequestParser rp(ctx, 0x83A, 4, 0);
+    MediaType media_type = static_cast<MediaType>(rp.Pop<u8>());
```

Comment:
```suggestion
    const auto media_type = static_cast<MediaType>(rp.Pop<u8>());
```

similar changes can be made to the two variables below this one.